server/hedley/modules/custom/hedley_case_management/hedley_case_management.module

```diff
@@ -361,14 +368,17 @@ function hedley_case_management_handle_tuberculosis_insert_trigger($node) {
 /**
  * Marks tuberculosis follow-ups belonging to person as resolved.
  *
+ * Also, marks acute illness follow-ups belonging to person as resolved,
+ * in case their diagnose is TB suspect.
+ *
  * @param int $person_id
  *   Person ID.
  * @param int $to_nid
  *   Optional; IDs of loaded follow-ups should be below this node ID.
  * @param int $limit
  *   Maximal number of follow-ups to load. Defaults to 200.
  */
-function hedley_case_management_resolve_tuberculosis_follow_ups($person_id, $to_nid = NULL, $limit = 200) {
+function hedley_case_management_resolve_tuberculosis_follow_ups($person_id, $to_nid = NULL, $limit = 1000) {
```

Comment:
Why did we need to bump the limit from 200 to 1000? Were there such cases with more than 200 follow-ups? It sounds suspicious at first sight.

Reply:
Well, no. Person should not have more than 200 follow  ups, specially, that follow ups are getting resolved, and excluded from query results. So I'll revert the change.